src/tooltip.tsx

```diff
@@ -135,10 +140,41 @@ const Tooltip: React.FC<Props> = ({
         height: 0,
     });
 
-    const getPosition = (position: Position = defaultPositionDesktop) =>
-        isTabletOrSmaller && (position === 'left' || position === 'right') ? defaultPositionMobile : position;
+    const [containerPosition, setContainerPosition] = React.useState({});
 
-    const position = getPosition(rest.position);
+    const getPosition = (position: Position | undefined) => {
+        let finalPosition = position ? position : defaultPositionDesktop;
+        if (isTabletOrSmaller) {
+            finalPosition = position ? position : defaultPositionMobile;
+        }
+
+        return finalPosition;
+    };
```

Comment:
Maybe a const? and make a short handed if for the isTabletOrSmaller

`const finalPosition = isTabletOrSmaller ? (position || defaultPositionMobile) : (position || defaultPositionDesktop);`

Reply:
Looking here again, we can use:

`return isTabletOrSmaller ? (position || defaultPositionMobile) : (position || defaultPositionDesktop);`

There is no need for the finalPosition const